packages/perseus/src/widgets/numeric-input/utils.ts

```diff
@@ -75,47 +87,3 @@ export const shouldShowExamples = (
 
     return !allFormsAccepted;
 };
-
-/**
- * uniqueAnswerForms takes a list of answer forms and returns a list of unique
- * answer forms. This is useful for ensuring that we don't show duplicate examples
- * to the user.
- */
-const getUniqueAnswerForms = function (
```

Comment:
These were moved to `perseus-core` because I needed them for deriving answer forms.